src/components/BreadcrumbMenu/utils.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { useParams, matchPath } from 'react-router-dom';
+
+const DynamicBreadcrumb = () => {
+  const { id } = useParams();
+  return <span>{id}</span>;
+};
+
+export const routes = [
+  { path: '/', breadcrumb: 'Home' },
+  { path: '/admin', breadcrumb: null },
+  { path: '/admin/cohorts/:id', breadcrumb: DynamicBreadcrumb },
+];
+
+const excludePaths = ['/', '/admin/cohorts/:id'];
+
+export const isRouteExcluded = (pathname: string) => {
+  return excludePaths.find((path) => matchPath({ path }, pathname));
+};
```

Comment:
I'm not familiar with React breadcrumbs, but is the '/' needed? Since we don't have '/' path for admin.

Reply:
The '/' was a cosmetic choice to split up the breadcrumbs so they weren't run together. Without it, the output is more like "AdminCohorts1Dashboard". I wonder if we could use a space instead?